scene2d/src/test/kotlin/ktx/scene2d/factoryTest.kt

```diff
@@ -6,10 +6,7 @@ import com.badlogic.gdx.scenes.scene2d.ui.Cell
 import com.badlogic.gdx.scenes.scene2d.ui.Label
 import com.badlogic.gdx.scenes.scene2d.ui.Tree.Node
 import com.kotcrab.vis.ui.VisUI
-import org.junit.Assert
-import org.junit.Assert.assertEquals
-import org.junit.Assert.assertNotNull
-import org.junit.Assert.assertTrue
+import org.junit.Assert.*
```

Comment:
Please don't use wildcard imports, thanks.

Reply:
Will fix that too. This is done automatically for me I think when more than 5 imports of the same package are imported. Then IntelliJ makes an asterisk import for me.

Is this something that can be configured in .editorconfig file of the project?

Reply:
I think editor config is more language-agnostic than than, but I'd have to check.